.github/CODEOWNERS

```diff
@@ -78,7 +78,7 @@
 
 # > Service: ARM - Core
 # ServiceLabel: %ARM - Core %Service Attention
-#/<NotInRepo>/            @armleads-azure
+/sdk/core/Azure.Core           @armleads-azure
```

Comment:
Azure.Core is not the right one for this one.

Reply:
Ah okay my bad - we can leave it as NotInRepo instead then